wagtailnhsukfrontend/settings/models.py

```diff
@@ -1,24 +1,12 @@
 from django.db import models
 from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel
-from wagtail import VERSION as WAGTAIL_VERSION
+from wagtail.admin.panels import FieldPanel
+from wagtail.admin.panels import FieldPanel as ImageChooserPanel
+from wagtail.admin.panels import InlinePanel, MultiFieldPanel, PageChooserPanel
 from wagtail.contrib.settings.models import BaseSetting, register_setting
```

Comment:
```suggestion
from wagtail.contrib.settings.models import BaseSiteSetting, register_setting
```

Reply:
This will need to be update in the `HeaderSettings` & `FooterSettings` classes as well

Reply:
Applied the change to `HeaderSettings` & `FooterSettings` classes.